{"cve": {"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2019-12761", "ASSIGNER": "cve@mitre.org"}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-94"}]}]}, "references": {"reference_data": [{"url": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba", "name": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba", "refsource": "", "tags": ["Exploit", "Third Party Advisory"]}, {"url": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba", "name": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba", "refsource": "", "tags": ["Exploit", "Third Party Advisory"]}, {"url": "https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html", "name": "[debian-lts-announce] 20190616 [SECURITY] [DLA 1819-1] pyxdg security update", "refsource": "", "tags": []}, {"url": "https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html", "name": "[debian-lts-announce] 20190616 [SECURITY] [DLA 1819-1] pyxdg security update", "refsource": "", "tags": []}, {"url": "https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html", "name": "[debian-lts-announce] 20210803 [SECURITY] [DLA 2727-1] pyxdg security update", "refsource": "", "tags": []}, {"url": "https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html", "name": "[debian-lts-announce] 20210803 [SECURITY] [DLA 2727-1] pyxdg security update", "refsource": "", "tags": []}, {"url": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562", "name": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562", "name": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562", "refsource": "", "tags": ["Third Party Advisory"]}]}, "description": {"description_data": [{"lang": "en", "value": "A code injection issue was discovered in PyXDG before 0.26 via crafted Python code in a Category element of a Menu XML document in a .menu file. XDG_CONFIG_DIRS must be set up to trigger xdg.Menu.parse parsing within the directory containing this file. This is due to a lack of sanitization in xdg/Menu.py before an eval call."}]}}, "configurations": {"CVE_data_version": "4.0", "nodes": [{"operator": "OR", "children": [], "cpe_match": [{"vulnerable": true, "cpe23Uri": "cpe:2.3:a:python:pyxdg:*:*:*:*:*:*:*:*", "versionEndExcluding": "0.26", "cpe_name": []}]}]}, "impact": {"baseMetricV3": {"cvssV3": {"version": "3.0", "vectorString": "CVSS:3.0/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:H", "attackVector": "NETWORK", "attackComplexity": "HIGH", "privilegesRequired": "NONE", "userInteraction": "REQUIRED", "scope": "UNCHANGED", "confidentialityImpact": "HIGH", "integrityImpact": "HIGH", "availabilityImpact": "HIGH", "baseScore": 7.5, "baseSeverity": "HIGH"}, "exploitabilityScore": 1.6, "impactScore": 5.9}, "baseMetricV2": {"cvssV2": {"version": "2.0", "vectorString": "AV:N/AC:H/Au:N/C:P/I:P/A:P", "accessVector": "NETWORK", "accessComplexity": "HIGH", "authentication": "NONE", "confidentialityImpact": "PARTIAL", "integrityImpact": "PARTIAL", "availabilityImpact": "PARTIAL", "baseScore": 5.1}, "severity": "MEDIUM", "exploitabilityScore": 4.9, "impactScore": 6.4, "acInsufInfo": false, "obtainAllPrivilege": false, "obtainUserPrivilege": false, "obtainOtherPrivilege": false, "userInteractionRequired": true}}, "publishedDate": "2019-06-06T19:29Z", "lastModifiedDate": "2024-11-21T04:23Z"}